UnionComponents/Shared/Option.razor

```diff
@@ -1,14 +1,15 @@
-﻿@typeparam TValue
-@OptionValue.Match(Some: x => Some, None: () => None)
+@typeparam TValue
+@optionValue.Match(Some: x => Some, None: () => None)
 
 @code
 {
     [Parameter]
-    public LanguageExt.Option<TValue> OptionValue { get; set; }
+    public TValue OptionValue { get; set; }
```

Comment:
I think the type should just be `Option<TValue>`

Reply:
I don't think it should be, largely because the calling component shouldn't need to know about `Option<T>`